pallets/integration-tests/src/tests/mnt_token_tests.rs

```diff
@@ -172,15 +177,125 @@ mod tests {
+
+				// ALice deposits DOT and enables her DOT pool as a collateral.
+				assert_ok!(MinterestProtocol::deposit_underlying(alice(), DOT, ONE_HUNDRED));
+				assert_ok!(MinterestProtocol::enable_is_collateral(alice(), DOT));
+
+				set_block_number_and_refresh_speeds(20);
+
+				assert_ok!(MinterestProtocol::borrow(alice(), ETH, 50_000 * DOLLARS));
+
+				// Accrued MNT tokens are equal to zero, since distribution occurs only at
+				// the moment of repeated user interaction with the protocol
+				// (deposit, redeem, borrow, repay, transfer, claim).
+				assert_eq!(TestMntToken::mnt_accrued(ALICE), Balance::zero());
+				assert_eq!(Tokens::free_balance(MNT, &ALICE), Balance::zero());
+
+				set_block_number_and_refresh_speeds(30);
+
+				// There are borrow in all pool, but BTC pool excluded from MNT distribution.
+				test_mnt_speeds(33_333_333_283_333_335, 66_666_666_716_666_664, 0);
+
+				// Init BTC pool
+				assert_ok!(MinterestProtocol::create_pool(
+					admin(),
+					BTC,
+					Rate::saturating_from_rational(8, 10),
+					Rate::zero(),
+					Rate::saturating_from_rational(9, 1_000_000_000),
+					Rate::saturating_from_rational(207, 1_000_000_000),
+					Rate::saturating_from_rational(1, 10),
+					Rate::saturating_from_rational(5, 1000),
+					Rate::saturating_from_rational(9, 10),
+					PROTOCOL_INTEREST_TRANSFER_THRESHOLD,
+					Rate::saturating_from_rational(1, 10),
+					Rate::saturating_from_rational(2, 10),
+				));
+				assert_ok!(MinterestProtocol::deposit_underlying(admin(), BTC, ONE_HUNDRED));
+				assert_ok!(MinterestProtocol::enable_is_collateral(admin(), BTC));
+				assert_ok!(MinterestProtocol::borrow(admin(), BTC, 50_000 * DOLLARS));
+
+				assert_ok!(MinterestProtocol::borrow(alice(), BTC, 30_000 * DOLLARS));
+
+				set_block_number_and_refresh_speeds(70);
+
+				// The BTC pool is excluded from the MNT token distribution, so its speed is zero.
+				test_mnt_speeds(33_333_333_283_333_335, 66_666_666_716_666_664, 0);
+				assert_eq!(TestMntToken::mnt_accrued(ALICE), Balance::zero());
+				assert_eq!(TestMntToken::mnt_accrued(BOB), Balance::zero());
+
+				assert_ok!(TestMntToken::enable_mnt_minting(admin(), BTC));
+				test_mnt_speeds(21_739_130_719_754_245, 43_478_261_537_334_575, 34_782_607_742_911_179);
+				assert_eq!(TestMntToken::mnt_accrued(ALICE), Balance::zero());
+				assert_eq!(TestMntToken::mnt_accrued(BOB), Balance::zero());
+
+				assert_ok!(MinterestProtocol::repay_all(alice(), ETH));
+
+				set_block_number_and_refresh_speeds(80);
+
+				test_mnt_speeds(27_777_711_264_044_877, 27_777_952_513_478_936, 44_444_336_222_476_186);
+				assert_eq!(TestMntToken::mnt_accrued(ALICE), 1_583_333_261_583_256_134);
+				assert_eq!(TestMntToken::mnt_accrued(BOB), Balance::zero());
+
+				// Alice шы able to claim rewards from all three pools
+				assert_ok!(MinterestProtocol::claim_mnt(alice(), vec![DOT, ETH, BTC]));
+				assert_eq!(Currencies::free_balance(MNT, &ALICE), 2_989_130_353_325_167_984);
```

Comment:
To make sure that alice accepted rewards from BTC pool
```suggestion
				assert_ok!(MinterestProtocol::claim_mnt(alice(), vec![DOT, ETH]));
				assert_eq!(Currencies::free_balance(MNT, &ALICE), 2858695574289277984);
				assert_ok!(MinterestProtocol::claim_mnt(alice(), vec![BTC]));
				assert_eq!(Currencies::free_balance(MNT, &ALICE), 2989130353325167984);
			})

```

Reply:
Done